package.json

```diff
@@ -76,8 +76,8 @@
     "test:unit:onetest": "test -z $CI && NODE_OPTIONS=--experimental-vm-modules jest --env=jsdom --coverage --colors --testPathPattern=\"src/app/components/\" || NODE_OPTIONS=--experimental-vm-modules jest --ci --runInBand --env=jsdom --coverage --colors --testPathPattern=\"src/app/components/\"",
     "test:unit:watch": "yarn test:unit -- --watch",
     "test:unit:updatesnapshots": "NODE_OPTIONS=--experimental-vm-modules jest --updateSnapshot --env=jsdom --coverage --silent=false --colors --testPathIgnorePatterns=\"src/integration\" \"scripts/bundleSize\" \"puppeteer\" \"ws-nextjs-app\"",
-    "test:integration": "node src/integration/utils/runTests/index.js",
-    "test:integration:updatesnapshots": "node src/integration/utils/runTests/index.js  --updateSnapshot",
+    "test:integration": "cp envConfig/test.env .env && node src/integration/utils/runTests/index.js",
```

Comment:
Are these copy commands needed?

Reply:
Heya, it's because the live pages have that switch will renders the old LegacyMediaPlayer if it's on live.

Reply:
Oh I see, it's already on test.